plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -37,7 +37,7 @@ import Flat.Decoder
 import Flat.Encoder as Flat
 
 -- See Note [Pattern matching on built-in types].
--- TODO: should we have the commonest builtins at the front to have more compact encoding?
+-- TODO: should we have the commonest built-in functions at the front to have more compact encoding?
```

Comment:
I want to be maximally unambiguous in this module, hence I don't use the "builtins" term here.